src/message-fragmentation.md

```diff
@@ -6,16 +6,22 @@ destination node is responsible for collecting the fragments together and reasse
 
 ## Fragment structure
 
+// TODO it is not exactly clear where this fragment is when looking at `sphinx`. IIUC sphinx payload contains
+// the fragment. (means that some of the point I raised in `sphinx.md` can just refer to this md.
 The 2,048 bytes in a fragment are split as follows:
 
 | Bytes    | Field                   | Description                                                    |
 |----------|-------------------------|----------------------------------------------------------------|
 | 0..15    | `message_id`            | The same for all fragments of a message                        |
 | 16..17   | `num_fragments_minus_1` | Little-endian number of fragments minus one                    |
+// TODO theorical max size of a full message (2^16 + 1) * fragement payload size.
+// TODO with two byte of size, is it really worth it to minus 1, I mean it prevents size 0, but size 0 is not
+// that problematic, it would just mean empty message.
 | 18..19   | `fragment_index`        | Little endian index of this fragment                           |
 | 20..21   | `data_size`             | Little-endian number of bytes of message data in this fragment |
 | 22       | `num_surbs`             | Number of SURBs in this fragment                               |
 | 23..2047 | `payload`               | Message data and SURBs                                         |
+// TODO explicitely state if surbs are before or after message.
```

Comment:
The paragraph below is explicit about this I think?

Reply:
yes I did understand after reading the whole spec, (not the paragraph below but the later parts), having it stated before would have facilitate my reading, but really not needed indeed.